### ISSUE TITLE
README: describe use with niv

### ISSUE THREAD

**Issue by siddharthist** [nmattia/snack#119]
It's not immediately clear to me how to use `snack` in conjunction with `niv` (or if they can be used together). Would you mind giving an example in the README?

**Comment by Skyfold** [comment]
It looks like (from my brief play with `niv`) that it simplifies pinning to a nixpkgs branch and adding external resources to your project. For example, the `snack` repository uses `niv` to pin its nixpkgs and bring niv in as an external dependency. However, `niv` does not hook those dependencies into a `shell.nix` file, instead you have to add some boilerplate to make importing those dependencies into a `shell.nix` easy. The `snack` repository itself is a good example of a working example of `niv` and `snack`.

tl;dr:
```
nix-env -iA niv -f https://github.com/nmattia/niv/tarball/master (install niv)
cd <your project>
niv init
niv update nixpkgs -b nixos-19.03 (if you want nixos-19.03 intsead of 18.09)
niv add nmattia/snack
(import ./nix/sources.nix directly into your shell.nix or do it like in snack's repo)
```
Then you have niv and snack pinned with a specific nixpkgs

**Comment by nmattia** [comment]
@siddharthist having `niv` install instructions in the README is a great idea! I'll do this asap.
